kalite/topic_tools/content_models.py

```diff
@@ -281,7 +286,9 @@ def get_topic_update_nodes(parent=None, **kwargs):
             Item.total_files,
             Item.id,
             Item.path,
-        ).join(Parent, on=(Item.parent == Parent.pk)).where((selector) & (Item.total_files != 0))
+        ).join(Parent, on=(Item.parent == Parent.pk)).where((selector))
+        # Not filtering on total_files != 0
+        # ).join(Parent, on=(Item.parent == Parent.pk)).where((selector) & (Item.total_files != 0))
```

Comment:
@rtibbles @aronasorman this is possibly a hotfix or something you need to fix in the upstream content packs. Running management command `annotate_content_items` does not fix it.